ex02/ex02.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+#
+#recursive search script
+
+for pattern in $@; do
+	path=$(ls -R ./ | egrep $pattern)
```

Comment:
the space between symbols is required. Please note it for the future
```suggestion
	path = $(ls -R ./ | egrep $pattern)
```